apps/automation-v2/src/config/prodnets.ts

```diff
@@ -9,7 +9,7 @@ const TLC_BATCH_LIQUIDATE_CONFIG: TlcBatchLiquidateConfig = {
   ACC_LIQ_MAX_CHUNK_NO: 100,
   MIN_ETH_BALANCE_WARNING: parseEther('0.1'),
   GAS_LIMIT: 1_000_000n,
-  SUBGRAPH_URL: 'https://api.thegraph.com/subgraphs/name/medariox/v2-mainnet',
+  SUBGRAPH_URL: 'https://api.thegraph.com/subgraphs/name/templedao/templedao-v2-mainnet',
```

Comment:
@medariox was this ready to go and in sync now?

If so, @frost-ostrich could you redeploy the TLC liquidation bot with this latest once merged?

Reply:
Yep, it's in sync. Just waiting for frost-ostrich's changes to merge.

Reply:
Let's merge this, and then I'll redeploy the TLC liquidation bot, I don't need to do any src change here